lms/djangoapps/instructor/tests/views/test_instructor_dashboard.py

```diff
@@ -106,6 +120,20 @@ def has_instructor_tab(user, course):
         student = UserFactory.create()
         self.assertFalse(has_instructor_tab(student, self.course))
 
+    def test_instructor_course_info(self):
+        """
+        Verify that it shows the correct course information
+        """
+        response = self.client.get(self.course_info_url)
+        self.assertIn("course-display-name", response.content)
+        self.assertIn("How to defeat the Road Runner", response.content)
+        self.assertIn("course-name", response.content)
+        self.assertIn("2017", response.content)
+        self.assertIn("course-number", response.content)
+        self.assertIn("001", response.content)
+        self.assertIn("course-organization", response.content)
+        self.assertIn("ACME", response.content)
```

Comment:
@jagonzalr Can you use PyQuery here like you did with https://github.com/edx/edx-platform/pull/16671, and ensure that the correct course elements are showing where they should be?

Also, since this change is mostly about using the proper "display" versions of the course fields, can you please add a test for that as well?

Reply:
Will do @pomegranited :)